run_robot_test.sh

```diff
@@ -43,6 +43,6 @@ ${VENV_ROOT}/bin/pip install -r requirements.txt
 #run tests
 tmp_dir=$(mktemp -d -t ods-ci-$(date +%Y-%m-%d-%H-%M)-XXXXXXXXXX)
 mkdir $tmp_dir
-./venv/bin/robot -d $tmp_dir --variable CONSOLE_URL=$CONSOLE_URL --variable KUBEADMIN=$KUBEADMIN --variable KUBEPASSWD=$KUBEPWD tests/Tests/test.robot
+./venv/bin/robot -d $tmp_dir -x xunit_test_result.xml -r test_report.html --variable CONSOLE_URL=$CONSOLE_URL --variable KUBEADMIN=$KUBEADMIN --variable KUBEPASSWD=$KUBEPWD tests/Tests/test.robot
```

Comment:
can you prefix $tmp_dir/ to the -r and -x as well, Thanks

Reply:
Hi Vasu,

By default, the command prefixes $tmp_dir to -r and -x values. For example,
# python3 -m robot -d /tmp -x test_result.xml -r report.html test.robot
==============================================================================
Test                                                                          
==============================================================================
My First Robot Test                                                   Hello Robot World!
My First Robot Test                                                   | PASS |
------------------------------------------------------------------------------
Test                                                                  | PASS |
1 critical test, 1 passed, 0 failed
1 test total, 1 passed, 0 failed
==============================================================================
Output:  /tmp/output.xml
XUnit:   /tmp/test_result.xml
Log:     /tmp/log.html
Report:  /tmp/report.html

Reply:
Thanks